src/ui/component/dropDownMenu/dropDownMenu.scss

```diff
@@ -0,0 +1,157 @@
+@import '@/ui/style/fonts.scss';
+@import '@/ui/style/theme.scss';
+
+.okp4-dataverse-portal-drop-down-menu-main {
+  @include with-theme {
+    .okp4-dataverse-portal-drop-down-menu-blur {
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+      backdrop-filter: blur(1px);
+      /* stylelint-disable-next-line property-no-vendor-prefix */
+      -webkit-backdrop-filter: blur(1px);
+    }
+
+    .okp4-dataverse-portal-drop-down-menu-container {
+      width: 100%;
+      position: relative;
+
+      .okp4-dataverse-portal-drop-down-menu-item {
+        @include noto-sans(700);
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding: 10px 18px;
+        color: themed('text');
+        cursor: pointer;
+        border-radius: 6px;
+        user-select: none;
+        /* stylelint-disable property-no-vendor-prefix */
+        -webkit-user-select: none;
+        -ms-user-select: none;
+        -moz-user-select: none;
```

Comment:
I think it's a good idea to prevent some inconsistencies from one browser to another, remains to be seen if the weight is acceptable @fredericvilcot ? 
For example, we now have these plugins: htmlConfig: 17.5kB, svgr plugin: 8.67kB , react swc plugin: 31.5kB.
Autoprefixer is 199 kB.